File: GenerateHashes.java
import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;

public class GenerateHashes {
    public static void main(String[] args) {
        BCryptPasswordEncoder encoder = new BCryptPasswordEncoder();
        String[] passwords = {"User123!", "Coach123!", "Desk123!", "Facility123!", "Admin123!", "Observer123!"};
        String[] usernames = {"test.user", "test.coach", "test.desk", "test.facility.admin", "test.sport.director", "test.ministry.observer"};
        
        for (int i = 0; i < passwords.length; i++) {
            System.out.println(usernames[i] + ": " + encoder.encode(passwords[i]));
        }
    }
}

File: generate_hashes.py
import bcrypt

passwords = {
    'test.user': 'User123!',
    'test.coach': 'Coach123!',
    'test.desk': 'Desk123!',
    'test.facility.admin': 'Facility123!',
    'test.sport.director': 'Admin123!',
    'test.ministry.observer': 'Observer123!'
}

for username, pw in passwords.items():
    hashed = bcrypt.hashpw(pw.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
    print(f"{username}: {hashed}")
